controllers/operands/kubevirt.go

```diff
@@ -454,6 +459,47 @@ func hcLiveMigrationToKv(lm hcov1beta1.LiveMigrationConfigurations) (*kubevirtco
 	}, nil
 }
 
+func hcTLSProtocolVersionToKv(profileMinVersion openshiftconfigv1.TLSProtocolVersion) kubevirtcorev1.TLSProtocolVersion {
+	switch profileMinVersion {
+	case openshiftconfigv1.VersionTLS10:
+		return kubevirtcorev1.VersionTLS10
+
+	case openshiftconfigv1.VersionTLS11:
+		return kubevirtcorev1.VersionTLS11
+
+	case openshiftconfigv1.VersionTLS12:
+		return kubevirtcorev1.VersionTLS12
+
+	case openshiftconfigv1.VersionTLS13:
+		return kubevirtcorev1.VersionTLS13
+
+	default:
+		return kubevirtcorev1.VersionTLS12
+	}
+}
+
+func hcTLSSecurityProfileToKv(profile *openshiftconfigv1.TLSSecurityProfile) *kubevirtcorev1.TLSConfiguration {
+	var profileCiphers []string
+	var profileMinVersion openshiftconfigv1.TLSProtocolVersion
+
+	if profile == nil {
+		return nil
+	}
+
+	if profile.Custom != nil {
+		profileCiphers = profile.Custom.TLSProfileSpec.Ciphers
```

Comment:
Is there a check that prevents to specify ciphers when using 1.3 as `minTLSVersion`?
1.3 ciphers will be skipped by `crypto.OpenSSLToIANACipherSuites(profileCiphers)` and this is ok;
but what happens if a user uses a wrong configuration? i.e. if I specify 
```
minTLSVersion: VersionTLS13,
ciphers: <one compatible tls1.2 cipher>
```
this will result in a reject from kv webhook.
More in general, the question is: how hco handle wrong custom configurations?
Thanks

Reply:
HCO webhook is always going to try validating each change with all the components in dry-run mode, so each configuration change will be accepted only if all the components are going to accept it.
On the other side, if kv webhook will refuse the change, HCO webhook will refuse as a consequence.

Reply:
Thanks for the clear explanation. In the light of that, it seems to me that everything is great! How about adding a test for this? WDYT? Or is it already tested in another place? Thanks

Reply:
Not on this specific failure from kw webhook, but in general the mechanism is already tested in validator_test.go,
see: `should return error if dry-run update of KV CR returns error`

Reply:
Sounds great! No other objection. Thanks